datageneration/relation_generator.py

```diff
@@ -113,21 +113,14 @@ def generate_relation_with_contain(self, area_entities: List[Entity], point_enti
             relations.extend(self.generate_individual_distances(other_entity_ids))
             relation_type = "individual_distances_with_contains"
         else:
-            # Only assign "contains within radius" if there is only one contains group and more than one point entity
-            if np.random.choice([True, False], p=[self.ratio_within_radius_within,
-                       1 - self.ratio_within_radius_within]) and len(point_entities_connecting_to_area_entity[0]) > 1:
-                relations = self.generate_relation_with_contain_helper(drawn_area_entities,
-                                                   point_entities_connecting_to_area_entity, True)
-                relation_type = "contains_within_radius"
-            else:
-                relations = self.generate_relation_with_contain_helper(drawn_area_entities,
-                                                   point_entities_connecting_to_area_entity)
-                relation_type = "contains_relation"
+            relations = self.generate_relation_with_contain_helper(drawn_area_entities,
+                                               point_entities_connecting_to_area_entity)
+            relation_type = "contains_relation"
```

Comment:
we use sometimes "contains" and sometimes "contains_relation". Is there difference?  If not perhaps you can define it as an [ENUM parameter](https://github.com/dw-innovation/kid2-spot-datageneration/blob/031ed96ec2ce0d4959b982931645b6da97c316c2/datageneration/relation_generator.py#L10)  for avoiding confusion.

Reply:
There are two "type" definition, one for the individual Relation objects, and one for the parent Relations.